server/api/src/rs_get_rescinfo_for_get.cpp

```diff
@@ -0,0 +1,94 @@
+#include <cstring> // for strdup.
+#include <nlohmann/json.hpp> // for strdup.
+
+//// ----------- TODO: make this file
+// #include "irods/rs_get_rescinfo_for_put.hpp"
+
+#include "irods/catalog_utilities.hpp"
+#include "irods/irods_logger.hpp"
+#include "irods/rodsErrorTable.h"
+
+#include "irods/getHostForPut.h" // for definition of macro: THIS_ADDRESS
+#include "irods/getHostForGet.h"
+#include "irods/rodsLog.h"
+#include "irods/rsGlobalExtern.hpp"
+#include "irods/rcGlobalExtern.h"
+#include "irods/getRemoteZoneResc.h"
+#include "irods/dataObjCreate.h"
+#include "irods/objMetaOpr.hpp"
+#include "irods/resource.hpp"
+#include "irods/collection.hpp"
+#include "irods/specColl.hpp"
+#include "irods/miscServerFunct.hpp"
+#include "irods/rsGetHostForGet.hpp"
+
+// =-=-=-=-=-=-=-
+#include "irods/irods_resource_backport.hpp"
+#include "irods/irods_resource_redirect.hpp"
+
+
+namespace
+{
+    using log_api = irods::experimental::log::api;
+} // anonymous namespace
+
+
+auto rs_get_rescinfo_for_get(RsComm* rsComm, dataObjInp_t *dataObjInp, char** _resp) -> int
```

Comment:
Consider whether it's worth moving this into the cpp file where `rsGetHostForGet` is defined.

Reply:
Guess we're ok with things as they stand, and the rs_* server function being in its own file ? I'm not clear on the conventions here.

Reply:
Ignore comments referencing old code.

Reply:
Ok, wasn't sure I could just ignore those, being as much of this old code didn't get deleted as such.  Sometimes it just got renamed (as in, one API replaced 2 others) or moved to other places.